quran/quran.com-frontend-next
# add Dropdown DLS, verse settings dropdown and copying verse functionality

### Summary
This PR adds non-advanced verse copying functionality and introduces `Dropdown` to QDC's DLS. A story of the `Dropdown` is also added.

### Test Plan
1. Go to the first chapter.
2. Click on a verse's settings icon.
3. A dropdown containing a menu that has a Copy item should appear.
4. Click on Copy.
5. The text of the verse should be copied into the clipboard and the text should change to "Copied!" instead of "Copy".
6. After 3 seconds, "Copied!" should be changed back to "Copy".

### Screenshots

| Before | After |
| ------ | ------ |
| ![Before](https://user-images.githubusercontent.com/15169499/126768926-8b0540ea-2209-4234-904f-d9eb32314eab.png)| ![Copy](https://user-images.githubusercontent.com/15169499/126768941-7b055d55-1d8c-43ed-91e7-1b9d66b0d231.png) |
| | ![Copied](https://user-images.githubusercontent.com/15169499/126768937-09f53b14-fc7d-4a63-adce-f5ab2048a6cb.png)|
